components/Author.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+
+import AuthorCollection from '../interfaces/AuthorCollection'
+import { MarkdownFileWithUrl } from '../pages/authors'
+
+import ContentCardList from './ContentCardList'
+import PageLayout from './PageLayout'
+
+export interface Props {
+ validAuthor: AuthorCollection
+ filteredRecordsWithUrl: MarkdownFileWithUrl[]
+}
```

Comment:
We could simplify the prop names. I think the type, `MarkdownFileWithUrl`, is enough to convey the information that the records need to have a url, so that's why I suggest we can remove `withUrl` from the prop name.
```suggestion
export interface Props {
 author: AuthorCollection
 records: MarkdownFileWithUrl[]
}
```